src/server/index.js

```diff
@@ -1,43 +1,18 @@
-import emailjs from "emailjs"
 import express from "express"
 import {assoc, map} from "ramda"
 import favicon from "serve-favicon"
 import path from "path"
+import winston from "winston"
 
 import handleRender from "./render"
 import {addNormalizedDescription} from "../search"
 import {fetchParameters, fetchVariables, fetchSwagger} from "../webservices"
-
-
-function sendErrorEmail(req, err) {
-  const server = emailjs.server.connect({
-    host: "localhost",
-  })
-  const fullUrl = req.protocol + "://" + req.get("host") + req.originalUrl
-  server.send(
-    {
-      from: "webmaster+legislation-explorer@openfisca.fr",
-      subject: `Legislation Explorer Error: ${err.toString()}`,
-      text: `
-URL: ${fullUrl}
-
-${err.stack}
-
-${JSON.stringify(req.headers, null, 2)}`,
-      to: "webmaster@openfisca.fr",
-    },
-    function(sendErr/*, message*/) {
-      if (sendErr) {
-        console.log("Error sending email:", sendErr)
-      } else {
-        console.log("Email sent")
-      }
-    }
-  )
-}
+import config from "../config"
 
 
 function startServer(state) {
+  winston.configure(config.winstonConfig);
```

Comment:
> Shouldn't this be put near the top of the file rather than in the `startServer` function? We'd like to log if something fails before `startServer` is called 🙂